common/static/coffee/src/discussion/views/discussion_content_view.coffee

```diff
@@ -1,12 +1,12 @@
 if Backbone?
   class @DiscussionContentView extends Backbone.View
 
-  
+
```

Comment:
I can remove this file from the diff if anyone really feels strongly, but fixing "whitespace on an empty line" seems to be an improvement.